src/ast/html_component.cr

```diff
@@ -1,13 +1,14 @@
 module Mint
   class Ast
     class HtmlComponent < Node
-      getter attributes, children, component, comments, ref
+      getter attributes, children, component, comments, ref, closing_tag_position
 
       def initialize(@attributes : Array(HtmlAttribute),
+                     @closing_tag_position : Int32?,
```

Comment:
It may be worth storing the closing tag as a node (maybe `TypeId` similar to `component` here) as it would make supporting "Go to definition" for it easier too 😄